dotnet/src/SemanticKernel.UnitTests/Orchestration/ContextVariablesTests.cs

```diff
@@ -176,7 +176,7 @@ public void SetWithoutLabelSucceeds()
         // Arrange
         string anyName = Guid.NewGuid().ToString();
         string anyContent = Guid.NewGuid().ToString();
-        ContextVariables target = new ContextVariables();
+        ContextVariables target = new();
```

Comment:
I'm curious if we want to prefer this or `var` usage instead.

Reply:
personally I prefer this syntax when possible, I find it useful having the type on the left side to set the context when reading the code

Reply:
Sounds good to me, maybe we can also update `csharp_style_deconstructed_variable_declaration` in editorconfig to be false then.